scripts/qa_crowbarsetup.sh

```diff
@@ -2837,7 +2893,7 @@ function onadmin_proposal
     # Check if there were any HA failures from the proposals so far
     safely oncontroller check_crm_failcounts
     # For all remaining proposals, check for HA failures after each deployment
-    for proposal in horizon ceilometer heat manila trove \
+    for proposal in ironic horizon ceilometer heat manila trove \
```

Comment:
I think ironic should be applied in the part "up through nova" a couple of lines above as it's logical order is just before nova.